src/50_Sample_AMPs/eCommerce_Sample.html

```diff
@@ -0,0 +1,168 @@
+  <!---{
+      "preview": true,
+       "experiment": true,
+       "component": "amp-social-share"
+    }--->
+  <!--
+    #### Introduction
+
+    This sample showcases how to use AMP for creating eCommerce page. Usually eCommerce page do contain rich contents such as images and product description. The key advantage of AMP eCommerce page is blazingly fast loading and no time rendering.
```

Comment:
I'm not sure whether we should call it "eCommerce page". The term eCommerce is too generic, maybe "product page"?